magefile.go

```diff
@@ -55,6 +55,7 @@ func init() {
 		"windows/386",
 		"windows/amd64",
 		"darwin/amd64",
+		"darwin/arm64",
```

Comment:
[For reviewers] Not sure about this since [v7 beats](https://github.com/elastic/beats/blob/v7.0.0/dev-tools/mage/platforms.go#L37) doesn't seem to support `Default` for `darwin/arm64`